pkg/dataplane/http/context.go

```diff
@@ -206,6 +208,11 @@ func (c *context) GetItem(getItemInput *v3io.GetItemInput,
 	return c.sendRequestToWorker(getItemInput, context, responseChan)
 }
 
+type attribValuesSection struct {
```

Comment:
Our Go convention is verbose variable naming. 
attribute not attrib and whatever acc means but verbose